packages/react-component-library/.storybook/preview-head.html

```diff
@@ -6,3 +6,10 @@
   .sbdocs-h2:first-of-type { margin-bottom: 40px; }
   .rn-sidebar--storybook { position: unset !important; max-height: 30rem; }
 </style>
+
+<script type="text/javascript">
```

Comment:
I wonder if there's a way to make it more obvious what this is for? Either separating it out or a comment 😄

Reply:
Added a HTML comment.